conf/defaults.ini

```diff
@@ -279,11 +279,11 @@ editors_can_admin = false
 # Login cookie name
 login_cookie_name = grafana_session
 
-# The lifetime (days) an authenticated user can be inactive before being required to login at next visit. Default is 7 days.
-login_maximum_inactive_lifetime_days = 7
+# The maximum lifetime (duration) an authenticated user can be inactive before being required to login at next visit. Default is 7 days (7d). This setting should be expressed as a duration, e.g. 5m (minutes), 6h (hours), 10d (days), 2w (weeks), 1M (month). The lifetime resets at each successful token rotation (token_rotation_interval_minutes).
+login_maximum_inactive_lifetime_duration = 
```

Comment:
Shouldn't the default values be reflected here? It now says the default is empty?

Reply:
It gets the default from the legacy/deprecated setting `login_maximum_inactive_lifetime_days`, if defined, or `7d`. See https://github.com/grafana/grafana/blob/5675496f6bd45c844e836eb277028358ed6d597c/pkg/setting/setting.go#L1233-L1240

Reply:
I know they get set to that value, but this is not reflected in this sample config file. For all other parameters the default value is shown in the commented parameter config.

Reply:
True, but it's reflected in the description of the parameter in the sample config file. Is this creating a problem, sorry I don't understand?

Reply:
There’s indeed no real problem. There’s just inconsistency introduced compared to all other settings.

Reply:
Thanks. I created #47509 and hopefully we get this sorted for Grafana v9.